lib/python/ensembl/brc4/runnable/manifest_stats.py

```diff
@@ -122,25 +125,21 @@ def get_seq_region_stats(self, seq_region_path):
 
         return stats
 
-    def get_json(self, json_path):
-        with open(json_path) as json_file:
-            return json.load(json_file)
-
-    def get_gff3_stats(self, gff3_path):
+    def get_gff3_stats(self, gff3_path: Path) -> List:
         
         stats = []
-        stats.append(os.path.basename(gff3_path))
-        if gff3_path.endswith(".gz"):
+        stats.append(gff3_path.name)
+        if gff3_path.name.endswith(".gz"):
```

Comment:
Not worth changing it, I believe, but you can also do: `gff3_path.suffix == ".gz"`. Furthermore, we could have a nice discussion as to which one would be more performant :grin:

Reply:
suffix is surely more efficient, but very likely insignificantly so :)